contents/docs/session-replay/_snippets/ios-installation.mdx

```diff
@@ -25,6 +25,14 @@ config.sessionReplayConfig.maskAllTextInputs = true
 // Whether images are masked. Default is true.
 config.sessionReplayConfig.maskAllImages = true
 
+// Enable masking of all sandboxed system views
```

Comment:
IIRC originated from an RN app right?
We'd need to sync those flags on RN SDK, session replay plugin and docs.
Happy to do it or guide you through them, what do you prefer?

Reply:
https://github.com/PostHog/posthog-react-native-session-replay/blob/e019a5ca4db64a29859ec87d8975c3d0cb47b37f/android/src/main/java/com/posthogreactnativesessionreplay/PosthogReactNativeSessionReplayModule.kt#L38-L39
https://github.com/PostHog/posthog-react-native-session-replay/blob/e019a5ca4db64a29859ec87d8975c3d0cb47b37f/ios/PosthogReactNativeSessionReplay.swift#L23-L27
https://github.com/PostHog/posthog-js-lite/blob/450f6a6f558472cdc91ea6b71206726e846763f3/posthog-react-native/src/posthog-rn.ts#L301-L302
https://github.com/PostHog/posthog-js-lite/blob/450f6a6f558472cdc91ea6b71206726e846763f3/posthog-react-native/src/types.ts#L51-L62
adding the new configs mentioning "iOS" only for now would be needed
then here https://posthog.com/docs/session-replay/installation?tab=React%20Native#step-two-configure-replay-settings

Reply:
since this is ios only, not for now